File: Stage9/C/Problem2581.c
#include <stdio.h>

int main(void) {
    int M, N;

    scanf("%d %d", &M, &N);

    int sum = 0, min = 0;

    for (int i = M; i <= N; i++) {
        if (i < 2) continue;

        int prime = 1;
        for (int j = 2; j * j <= i; j++) {
            if (i % j == 0) {
                prime = 0;
                break;
            }
        }

        if (prime) {
            sum += i;
            if (min == 0) min = i;
        }
    }

    if (sum == 0) {
        printf("-1\n");
    } else {
        printf("%d\n%d\n", sum, min);
    }

    return 0;
}

File: Stage9/Python/Problem2581.py
def is_prime(num):
    if num < 2:
        return False
    for i in range(2, int(num**0.5) + 1):
        if num % i == 0:
            return False
    return True

def main():
    M = int(input())
    N = int(input())
    
    sum = 0
    min = 0
    
    for i in range(M, N + 1):
        if is_prime(i):
            sum += i
            if min == 0:
                min = i
    if sum == 0:
        print(-1)
    else:
        print(sum)
        print(min)
        
if __name__ == "__main__":
    main()